src/controllers/user.js

```diff
@@ -83,3 +85,39 @@ export const updateById = async (req, res) => {
 
   return sendDataResponse(res, 200, { user: updatedUser })
 }
+
+export const deleteUserById = async (req, res) => {
+  const id = Number(req.params.id)
```

Comment:
What happens if this is not a number? We should probably create a response if that is the case